pkg/config/profile.go

```diff
@@ -333,6 +335,9 @@ func (p *Profile) writeProfile(runtimeViper *viper.Viper) error {
 		runtimeViper = p.safeRemove(runtimeViper, "publishable_key")
 	}
 
+	// Remove experimental fields during login
+	runtimeViper = p.removeExperimentalFields(runtimeViper)
+
```

Comment:
A bit orthogonal but there's been an unresolved bug in login where the config fields don't get cleared when you `stripe login`, so e.g. if you were to login to account A, then login to account B, some of account A's configs could persist if they're not set on account B (display_name being a common one for me). I wonder if we could just delete the entire profile on login.

Reply:
I think I ran into this exact bug you're talking about- removing individual config fields doesn't seem to always work. But removing a higher level umbrella field (in this case, `experimental`) **does** in all my testing (see other PR). That being said, if there's no reason to keep around data from the previous profile, i think removing the the entire thing is our safest bet

Reply:
Tested that [removing all experimental fields](https://github.com/stripe/stripe-cli/pull/1072/commits/6e8e5248d43b7ce1a7d0124d58e16678dc5101fc) works

@vcheung-stripe I also added a commit deleting the existing profile before creating a new one with the same name, let me know if you still want me to keep that in or revert it: https://github.com/stripe/stripe-cli/pull/1072/commits/ceeb41091807d243e7597b322b8fcc49914d6a1b

Reply:
sounds good, I'm okay with keeping that commit in since presumably it'd be buggy otherwise